packages/react-link/package.json

```diff
@@ -27,11 +27,9 @@
   },
   "devDependencies": {
     "@fluentui/a11y-testing": "^0.1.0",
-    "@fluentui/babel-make-styles": "9.0.0-beta.4",
     "@fluentui/eslint-plugin": "*",
-    "@fluentui/jest-serializer-make-styles": "9.0.0-beta.4",
```

Comment:
Don't we also need to add the replacements for these (@griffel/babel-preset and @griffel/jest-serializer)?